plugin/pkg/distributedlock/cassandra/lock.go

```diff
@@ -78,6 +79,20 @@ func (l *Lock) Acquire(resource string, ttl time.Duration) (bool, error) {
 	return false, nil
 }
 
+// Forfeit forfeits an existing lease around a given resource.
+func (l *Lock) Forfeit(resource string) (bool, error) {
+	var name, owner string
+	applied, err := l.session.Query(cqlDeleteLock, resource, l.tenantID).ScanCAS(&name, &owner)
+	if err != nil {
+		return false, errors.Wrap(err, "Failed to forfeit resource lock due to cassandra error")
+	}
+	if applied {
+		// The lock was successfully deleted
+		return true, nil
+	}
+	return false, errors.Wrap(errLockOwnership, "Failed to forfeit resource lock")
```

Comment:
We need a new error, this will return: "This host does not own the resource lock: Failed to foreit resource lock"